woocommerce/woocommerce
# Update order URLs for HPOS in e2e tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the URLs for order adding/editing in end-to-end tests for use with HPOS.

Closes https://github.com/woocommerce/woocommerce/issues/40846



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure CI passes
2. `pnpm env:restart && pnpm test:e2e-pw merchant`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
